pydantic/color.py

```diff
@@ -70,11 +70,13 @@ def __init__(self, value: ColorType) -> None:
             self._rgba = parse_tuple(value)
         elif isinstance(value, str):
             self._rgba = parse_str(value)
+        elif isinstance(value, Color):
+            self._rgba = value._rgba
```

Comment:
Maybe add `value = value._original` here?

Reply:
Added & removed the conditional below it - thank you for the suggest